<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расписание занятий СахГТИ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/files/main.png</normaloff>:/files/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6,2,1">
          <item>
           <widget class="QListWidget" name="grList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="predmList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="vidZanList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="kabList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="week_list">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="rasp">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <row>
         <property name="text">
          <string>1 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>5 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>6 пара</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>7 пара</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Новый столбец</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/files/update.png</normaloff>:/files/update.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить таблицу расписания</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/files/edit.png</normaloff>:/files/edit.png</iconset>
   </property>
   <property name="text">
    <string>Режим редактирования</string>
   </property>
   <property name="toolTip">
    <string>Включение или отключение режима редактирования расписания</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/files/delete.png</normaloff>:/files/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить занятие</string>
   </property>
   <property name="toolTip">
    <string>Удалить занятие</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
